### DISCUSSION TITLE
planner with a week on 2 pages

### DISCUSSION THREAD

**Discussion by Cockpitinferno** [kudrykv/latex-yearly-planner#100] [Ideas]
Hi every one, fisrt of all i would like to say a really BIG thank you for all these stuff, all work done!!

i tried pdf planners and there great but a bit too much for my need.
I use tab ultra C Pro for 1 month now and i would like to use a planner a little like my old paper planner.
[2023-sept-oct.pdf](https://github.com/kudrykv/latex-yearly-planner/files/13758463/2023-sept-oct.pdf)

see page 3 , and i precise there is no links, used it before to print it and put it in a A5 agenda.

a week on 2 pages and we on one case and notes on each day. ( i don't have enough space on the weeks pages that are porposed on the pdf planners wich are freely given)

I downloaded Mixtex and Go but before spending hours to understand how it works ( i'm not a coder at all and i'm french so even with a good english when it is technnical it's very difficult to understand) i would like to know if possible and not too difficult to build such a model.

i don't need daily pages etc... only calendar on each page and weeks but i need a little bit more space than with actual version.
if in french it's better but it is no priority.

Thank to every one who will be able to help and have happy christmas and happy new year for those who celebrate it.

**Comment by kudrykv** [comment]
Hi!
Given that you're not a developer, it likely might not be worth your while to tweak it. Disabling sections you don't need is easy — it's simply removing them from config files and generating the planner. It's more tricky with adjusting the template. Depending on how much you want to adjust, it can even require some code changes, not only the template change. For example, breaking a week template into two pages would require a template change and code change.

**Comment by Cockpitinferno** [comment]
Ok thank you for your answer.
i will try to find one already done.
